MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+
+        try
+        {
+            var _items = await table.GetItemsAsync();
+            Items.Clear();
+            foreach (var item in _items)
+            {
+                Items.Add(item);
+            }
+            IsBusy = false;
+        }
+        catch (Exception ex)
+        {
+            await Application.Current.MainPage.DisplayAlert("Error", ex.Message, "OK");
+        }
+    }
+
+    public event PropertyChangedEventHandler PropertyChanged;
+    public void OnPropertyChanged(string propertyName)
+    {
+        if (PropertyChanged == null)
+            return;
+
+        PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Refer to the following code which illustrates, how to bind the table contents into the `SfListView`.
+
+{% tabs %}
+{% highlight c# %}
+public partial class ToDoListPage : ContentPage
+{
+  public ToDoListPage(IEasyMobileServiceClient client)
+  {
+    InitializeComponent();
+    var viewModel = new ViewModels.ToDosViewModel(client);
+    BindingContext = viewModel;
+    listView.ItemsSource = viewModel.Items;
+  }
+  
+  private void FetchButton_Clicked(object sender, EventArgs e)
+  {
+    var viewModel1 = (ToDosViewModel)BindingContext;
+    viewModel1.RefreshCommand.Execute(null);
+  }
+}
+{% endhighlight %}
+
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.SfListView">
+<ContentPage.Content>
+        <StackLayout>
+            <syncfusion:SfListView x:Name="listView" SelectedItem="{Binding SelectedToDoItem, Mode=TwoWay}" ItemSize="50">
+            <syncfusion:SfListView.ItemTemplate>
+                    <DataTemplate>
+                        <ViewCell>
+                            <StackLayout Orientation="Horizontal" HorizontalOptions="FillAndExpand" VerticalOptions="CenterAndExpand">
+                                <Label Text="{Binding Text}"/>
+                                <Switch IsToggled="{Binding Completed}"/>
+                            </StackLayout>
+                        </ViewCell>
+                    </DataTemplate>
+                </syncfusion:SfListView.ItemTemplate>
+            </syncfusion:SfListView>
+            <Button Text="Add New" Command="{Binding AddNewItemCommand}"/>
+            <Button Text="Fetch" Command="{Binding FetchItemCommand}" />
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+
+## Loading data from SQLite offline database
+
+The `SfListView` allows binding the data from local database by using `SQLite`. To perform this, follow the steps:
+
+Step 1: Create a `SQLite database table`
+Step 2: Populate the data into the table
+Step 3: Store them as an `IEnumerable` collection
+Step 4: Bind it to SfListView using [SfListView.ItemSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemsSource) property

```

Comment:
Bind it to SfListView using [SfListView.ItemSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemsSource) property.